crates/fj-viewer/src/graphics/transform.rs

```diff
@@ -1,15 +1,15 @@
 use bytemuck::{Pod, Zeroable};
-use nalgebra::{Matrix4, Perspective3};
 
 use crate::camera::Camera;
+use fj_math::Transform as fjTransform;
```

Comment:
Maybe just remove this instead, and use it as `fj_math::Transform`? That's more verbose, but also more clear, I think.